app/_locales/en/messages.json

```diff
@@ -109,6 +109,9 @@
   "about": {
     "message": "About"
   },
+  "accept": {
+    "message": "Accept"
```

Comment:
Question: should we also add the corresponding text to other locales file?

Reply:
The locales/en/messages.json file is periodically translated and added to the other locale folders